src/extension/module.js

```diff
@@ -2337,7 +2342,29 @@ import sampleRUM from './rum.js';
             custom: true,
             id: id || `custom-plugin-${i}`,
             condition,
-            button: {
+            container: containerId,
+          };
+
+          if (isBadge) {
+            plugin.feature = true;
+            plugin.elements = [{
+              tag: 'span',
+              text: title,
+              attrs: {
+                class: 'hlx-sk-badge',
+              },
+            }];
+            // set badge styles
+            const isDarkSchemePreferred = () => window?.matchMedia?.('(prefers-color-scheme:dark)')?.matches ?? false;
```

Comment:
Does this work when the system scheme changes while the sidekick i open? i.e. are the colors updated automatically?

Reply:
Only after reload of the page/word, not automatically. Any better approach?

Reply:
I think you could inject an inline `style` element with the plugin's `elements` array, where you have the freedom to specify color scheme specific colors, e.g.
```css
.plugin.badge#id {
  color: black;
  background-color: red;
}

@media (prefers-color-scheme: dark) {
  .plugin.badge#id {
    color: white;
    background-color: blue;
  }
}
```